pkg/console/operator/status.go

```diff
@@ -235,96 +232,6 @@ func (c *consoleOperator) ConditionResourceSyncDegraded(operatorConfig *operator
-		Message:            "The operator is in an unmanaged state, therefore its availability is unknown.",
-		LastTransitionTime: metav1.Now(),
-	})
-	v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions, operatorsv1.OperatorCondition{
-		Type:               operatorsv1.OperatorStatusTypeProgressing,
-		Status:             operatorsv1.ConditionFalse,
-		Reason:             reasonUnmanaged,
-		Message:            "The operator is in an unmanaged state, therefore no changes are being applied.",
-		LastTransitionTime: metav1.Now(),
-	})
-	v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions, operatorsv1.OperatorCondition{
-		Type:               operatorsv1.OperatorStatusTypeDegraded,
-		Status:             operatorsv1.ConditionFalse,
-		Reason:             reasonUnmanaged,
-		Message:            "The operator is in an unmanaged state, therefore no operator actions are degraded.",
-		LastTransitionTime: metav1.Now(),
-	})
-
-	return operatorConfig
-}
-
-func (c *consoleOperator) ConditionsManagementStateRemoved(operatorConfig *operatorsv1.Console) *operatorsv1.Console {
-	v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions, operatorsv1.OperatorCondition{
-		Type: operatorsv1.OperatorStatusTypeAvailable,
-		// See https://github.com/openshift/api/pull/206
-		// At present, Available is the gate for upgrades.  The removal of an operand should NOT cause
-		// an upgrade to fail. Therefore, ManagementState:Removed should delete the operand (console),
-		// BUT should still report Available:True. Hopefully this will change.
-		Status:             operatorsv1.ConditionTrue,
-		Reason:             reasonRemoved,
-		Message:            "The operator is in a removed state, the console has been removed.",
-		LastTransitionTime: metav1.Now(),
-	})
-	v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions, operatorsv1.OperatorCondition{
-		Type:               operatorsv1.OperatorStatusTypeProgressing,
-		Status:             operatorsv1.ConditionFalse,
-		Reason:             reasonRemoved,
-		Message:            "The operator is in a removed state, therefore no changes are being applied.",
-		LastTransitionTime: metav1.Now(),
-	})
-	v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions, operatorsv1.OperatorCondition{
-		Type:               operatorsv1.OperatorStatusTypeDegraded,
-		Status:             operatorsv1.ConditionFalse,
-		Reason:             reasonRemoved,
-		Message:            "The operator is in a removed state, therefore no operator actions are degraded.",
-		LastTransitionTime: metav1.Now(),
-	})
-
-	return operatorConfig
-}
-
-// If ManagementState is invalid,
-// We don't know if the operand is available bc we exit the sync loop w/o assessing resources
-// We aren't progressing because we exit the sync loop
-// We don't know if we are degraded because we exit the sync loop
-func (c *consoleOperator) ConditionsManagementStateInvalid(operatorConfig *operatorsv1.Console) *operatorsv1.Console {
```

Comment:
Need to take care of https://github.com/openshift/console-operator/blob/master/pkg/console/operator/operator.go#L234